functions.py

```diff
@@ -0,0 +1,3 @@
+def sum(a, b):
```

Comment:
Now that I think about, please rename this function to `add`. Because python already has a `sum` function.

Reply:
Thanks for the suggestion, I have renamed the function to `add`.